tests/unit/util/async/AnyExecutionContextTests.cpp

```diff
@@ -270,6 +273,20 @@ TEST_F(AnyExecutionContextTests, TimerWithBoolHandlerExecuted)
     EXPECT_EQ(timer.get().value(), 42);
 }
 
+TEST_F(AnyExecutionContextTests, RepeatingOperation)
+{
+    auto mockRepeatingOp = RepeatingOperationType<std::any>{};
+    EXPECT_CALL(mockRepeatingOp, wait());
+    EXPECT_CALL(
+        mockExecutionContext, executeRepeatedly(An<std::chrono::milliseconds>(), An<std::function<std::any()>>())
```

Comment:
```suggestion
        mockExecutionContext, executeRepeatedly(A<std::chrono::milliseconds>(), A<std::function<std::any()>>())
```
Also why not 1 millisecond?